packages/upstream-extra/ocaml-lsp-server.1.4.1/opam

```diff
@@ -1,10 +1,7 @@
-x-commit-hash: "154ccd7142651807d40eb851969f318b873b7fb4"
```

Comment:
The version is not changed, are these changes fixing a problem?

The contents of the file are different from the default repo because I use `opam show --raw pkg.ver` to automate the package updates.